go.mod

```diff
@@ -2,9 +2,9 @@ module github.com/aws/amazon-cloudwatch-agent
 
 go 1.13
 
-replace github.com/influxdata/telegraf => github.com/aws/telegraf v0.10.2-0.20201015165757-4470de2d306b
+replace github.com/influxdata/telegraf => github.com/aws/telegraf v0.10.2-0.20220509174810-d4f1bd11c120
 
-replace github.com/shirou/gopsutil => github.com/aws/telegraf/patches/gopsutil v0.0.0-20201015165757-4470de2d306b // Temporary fix, pending PR https://github.com/shirou/gopsutil/pull/957
+replace github.com/shirou/gopsutil => github.com/aws/telegraf/patches/gopsutil v0.0.0-20220509174810-d4f1bd11c120 // Temporary fix, pending PR https://github.com/shirou/gopsutil/pull/957
```

Comment:
I'm really hesitant about making these changes. Let's leave these versions be where we can, and only upgrade things that are necessary to support the max dimension change (which I don't think requires any version bump)